ImageViewer/Source/ImageViewer/ImageViewer.swift

```diff
@@ -350,13 +350,13 @@ public final class ImageViewer: UIViewController, UIScrollViewDelegate, UIViewCo
             self.closeButton.alpha = 0.0
             self.view.transform = CGAffineTransformIdentity
             self.view.bounds = (self.applicationWindow?.bounds)!
-            self.imageView.frame = self.parentViewFrameInOurCoordinateSystem
+            self.imageView.frame = self.displacedView.frame
```

Comment:
The original approach does not work if the app or calling VC also supports landscape. In that case if the orientation is changed whilst the `ImageViewer` is showing, the value of `self.parentViewFrameInOurCoordinateSystem` is no longer correct. I've change the line to:

```self.imageView.frame = CGRectIntegral(self.applicationWindow!.convertRect(self.displacedView.bounds, fromView: self.displacedView))```

in fdcee13fc5702314145d11441d2f3a02c7da100a

which is recalculating the `parentViewFrameInOurCoordinates` as this this may have changed due to rotation. This works and addresses your concern.